src/app/pages/things/lora/lora.component.ts

```diff
@@ -203,14 +188,21 @@ export class LoraComponent implements OnInit {
           this.loraService.deleteDevice(event.data).subscribe(
             resp => {
               this.loraDevsNumber--;
-              this.getLoraChanNum();
             },
           );
         }
       },
     );
   }
 
+  searchLora(input) {
```

Comment:
This function gets repeated with exactly the same form. Maybe we need a service here.